comparison/ansible/packet_layer2.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - hosts: localhost
   tasks:
-    - name: Configure Packet switch for quad port intel machines
+    - name: Configure Eqinix Metal switch for quad port intel machines
```

Comment:
"Eqinix Metal" should be "Equinix Metal"

Reply:
```suggestion
    - name: Configure Equinix Metal switch for quad port intel machines
```